packages/open-next/src/adapters/cache.ts

```diff
@@ -298,7 +310,7 @@ export default class S3Cache {
     // If we use an in house version of getDerivedTags in build we should use it here instead of next's one
     const derivedTags: string[] =
       data?.kind === "FETCH"
-        ? data.data.tags ?? []
+        ? ctx?.tags ?? []
```

Comment:
Will this break backwards compatibility? Can we do `ctx?.tags ??data?.data?.tags ?? []

Reply:
Yeah you're right, that's how it should be done

Reply:
I just updated my PR with this change